src/main/scala/zio/prelude/GenFs.scala

```diff
@@ -59,6 +81,15 @@ object GenFs {
         Gens.nonEmptyListOf(gen)
     }
 
+  /**
+   * A generator of `Try` values.
+   */
+  val `try`: GenF[Random with Sized, Try] =
```

Comment:
I'm not sure about using ``try`` as name for this

Reply:
We try to avoid that. Maybe `tryScala`?

Reply:
Ok cool, sounds good!